EcalTiming/plugins/EcalTimingCalibProducer.cc

```diff
@@ -505,9 +519,10 @@ EcalTimingCalibProducer::Status EcalTimingCalibProducer::duringLoop(const edm::E
 #endif
 	// If we got less than the minimum recHits, continue
 	if(eventTimeMap_.size() < _recHitMin) return kContinue;
-	if(timeEB.num() < 4) return kContinue;
-	if(timeEEM.num() < 4 && timeEEP.num() < 4) return kContinue;
+	if(timeEB.num() + timeEEM.num() + timeEEP.num() < _minEntries) return kContinue;
```

Comment:
I agree (I didn't understand much the reason behind that, just changed to get the plots filled with low stat when one of the EEs was empty)